bots/tests-scan

```diff
@@ -131,17 +131,17 @@ def tests_invoke(priority, name, revision, ref, context, base, options):
     current = time.strftime('%Y%m%d-%H%M%M')
 
     checkout = "PRIORITY={priority:04d} bots/make-checkout --verbose"
-    cmd = "TEST_NAME='{name}-{current}' TEST_REVISION='{revision}' bots/tests-invoke"
+    cmd = "TEST_NAME={name}-{current} TEST_REVISION={revision} bots/tests-invoke"
```

Comment:
`current` is not currently passed through `pipes.quote`, but it's guaranteed to not have funny characters in it, right?

Reply:
Right, as it's the strftime from two lines up, so I don't think we need to bother with quoting there as it cannot be freely defined from outside. I  can add the quoting though, if you prefer?